SWLOR.Game.Server/Feature/ShipModuleDefinition/MissileLauncherModuleDefinition.cs

```diff
@@ -37,8 +38,13 @@ private void PerformAttack(uint activator, uint target, int dmg, int attackBonus
             var roll = Random.D100(1);
             var isHit = hitOverride ?? roll <= chanceToHit;
 
-            var attackerStat = GetAbilityScore(activator, AbilityType.Willpower);
-            var attack = Stat.GetAttack(activator, AbilityType.Willpower, SkillType.Piloting, attackBonus);
+            var attackerStat = GetAbilityScore(activator, AbilityType.Perception);
+            var attack = Stat.GetAttack(activator, AbilityType.Perception, SkillType.Piloting, attackBonus);
+            if (GetHasFeat(FeatType.IntuitivePiloting, activator) && GetAbilityScore(activator, AbilityType.Willpower) > GetAbilityScore(activator, AbilityType.Perception))
+            {
+                attackerStat = GetAbilityScore(activator, AbilityType.Willpower);
+                attack = Stat.GetAttack(activator, AbilityType.Willpower, SkillType.Piloting, attackBonus);
```

Comment:
I'd prefer you swap the variables in the if statement and only call Stat.GetAttack() once.